easybuild/easyconfigs/l/libpng/libpng-1.6.34-foss-2017b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.34'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
```

Comment:
missing `toolchainopts` line to enable building with `-fPIC`, see existing `libpng` easyconfigs